lib/dialects/postgres/index.js

```diff
@@ -74,7 +78,9 @@ class Client_PG extends Client {
       arrayAccessor = arrayAccessorMatch[2];
     }
 
-    return `"${value.replace(/"/g, '""')}"${arrayAccessor}`;
+    return isLiteralValue()
+      ? `'${value.replace(/"/g, '""')}'${arrayAccessor}`
```

Comment:
`wrapIdentifierImpl` method is implemented in each DBVender Class(ex. oracle, mysql, mssql) 

berows are other DB Class Implementations

[mssql - wrapIdentifierImpl](https://github.com/knex/knex/blob/176151d8048b2a7feeb89a3d649a5580786d4f4e/lib/dialects/mssql/index.js#L131) 

[mysql - wrapIdentifierImpl](https://github.com/knex/knex/blob/176151d8048b2a7feeb89a3d649a5580786d4f4e/lib/dialects/mysql/index.js#L59)

[sqlite3 - wrapIdentifierImpl](https://github.com/knex/knex/blob/176151d8048b2a7feeb89a3d649a5580786d4f4e/lib/dialects/sqlite3/index.js#L77)